pkg/resource/computed_attributes.go

```diff
@@ -26,9 +26,9 @@ func GenComputedAttributes(
 ) (property.Values, error) {
 	// Check.
 	if (entity.TemplateID == nil || !entity.TemplateID.Valid()) &&
-		(entity.ResourceDefinitionID == nil || !entity.ResourceDefinitionMatchingRuleID.Valid()) {
```

Comment:
Is it acceptable that ResourceDefinitionID is nil?

Reply:
ResourceDefinitionID isn't required for this function